.github/workflows/lfs-sync.yml

```
@@ -0,0 +1,198 @@
+                '',
+                'LFS-tracked files fetched from the contributor\'s fork and pushed to this repository\'s LFS storage. Verified with a fresh clone:',
+                '',
+                preview + extra,
+              ].join('\n'),
+            });
+
+      - name: Comment on PR — sync failed
+        if: steps.detect-lfs.outputs.has_lfs == 'true' && steps.verify.outputs.sync_ok == 'false'
+        uses: actions/github-script@v7
+        with:
+          script: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)